CHAID/__main__.py

```diff
@@ -43,7 +43,7 @@ def main():
         predictions.name = 'node_id'
         data = pd.concat([data, predictions], axis=1)
         print(data.to_csv())
-    elif nspace.predictions:
+    elif nspace.predict:
```

Comment:
Silly bug on the mainline. I've fixed it in master (without review soz lol), so a rebase will make this change go away.